models.py

```diff
@@ -41,3 +50,14 @@ class Skill:
     name: str
     proficiency: str
     logo: str
+
+
+@dataclass
+class Project:
+    '''
+    Project class
+    '''
+    name: str
+    language: str
```

Comment:
I'd say that multiple languages for a project might be better, as it'd support adding projects with multiple languages easily.